### ISSUE TITLE
db-extras-zynq7 writing files to wrong path

### ISSUE THREAD

**Issue by mithro** [SymbiFlow/prjxray#1050]
https://github.com/SymbiFlow/prjxray/blob/6d21194b569b2e320d7b1c4fe87c8db814a823b1/Makefile#L136-L140

Should be `database/zynq7/harness/` **not** `database/artix7/harness`.

**Comment by mithro** [comment]
```patch
diff --git a/Makefile b/Makefile
index 12be9a41..948fc7f0 100644
--- a/Makefile
+++ b/Makefile
@@ -137,7 +137,7 @@ db-extras-zynq7:
        +source minitests/roi_harness/zybo-swbut.sh && $(MAKE) -C fuzzers part_only
        +source minitests/roi_harness/zybo-swbut.sh && \
                $(MAKE) -C minitests/roi_harness \
-                       HARNESS_DIR=database/artix7/harness/zybo/swbut run
+                       HARNESS_DIR=database/zynq7/harness/zybo/swbut run
 
 db-check:
        @true
```

**Comment by tmichalak** [comment]
@mithro 
Created a PR with a fix (#1051), however I see you have applied the required changes for Zynq7 manually in the database.

**Comment by tmichalak** [comment]
The PR has been merged. The issue has been fixed.
